### PR TITLE
copy change from PR#3537 to all versions

### PR DESCRIPTION
**Pull request by sdwheeler**
Version(s) of document impacted
------------------------------
- [ ] Impacts 6.next document
- [ ] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [ ] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [x] This PR partially fixes the issue, and issue #3537 tracks the remaining work

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9319ca5](https://github.com/sdwheeler/PowerShell-Docs/pull/3598/commits/9319ca52e7e438005f69dec4e27737721667539c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 10 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9319ca5](https://github.com/sdwheeler/PowerShell-Docs/pull/3598/commits/9319ca52e7e438005f69dec4e27737721667539c)_: 

:clock10: **Incremental building**: average incremental building time is _13 min(s) 16 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[9319ca5](https://github.com/sdwheeler/PowerShell-Docs/pull/3598/commits/9319ca52e7e438005f69dec4e27737721667539c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/3.0/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-3.0&branch=pr-en-us-3598) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/4.0/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-4.0&branch=pr-en-us-3598) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/5.0/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-5.0&branch=pr-en-us-3598) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/5.1/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-5.1&branch=pr-en-us-3598) (powershell-5.1) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C24%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201901232347531970-3598%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Xn9mkFfYtak4fij199GzdClpjzkxZIrAzHr4wix4mPc%3D&st=2019-01-23T23%3A57%3A07Z&se=2019-02-24T00%3A02%3A07Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Management/Test-Path.md [hunk 1]
```diff
@@ -308,6 +308,17 @@ The acceptable values for this parameter are:
 
 Tells whether the final element in the path is of a particular type.
 
+> [!CAUTION] 
+> 
+> Up to PowerShell version 6.1.2, when the **IsValid** and **PathType** switches are 
+> specified together, the `Test-Path` cmdlet simply  the **PathType** switch and only
```
Comment (DCtheGeek) [inline reference/3.0/Microsoft.PowerShell.Management/Test-Path.md]: Looks like a word got removed between "simply" and "the **PathType**".  Also consider using a different word than simply.  Carry forward on versions.

**Comment by DCtheGeek** [comment]
@sdwheeler v6 doc should also be updated with the changes requested to keep the docs consistent, even though it was initially handled in the reference PR.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[5913798](https://github.com/sdwheeler/PowerShell-Docs/pull/3598/commits/59137988d56cadbdfe81f55ff06950452fbc3d04)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 15 sec(s)_

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[5913798](https://github.com/sdwheeler/PowerShell-Docs/pull/3598/commits/59137988d56cadbdfe81f55ff06950452fbc3d04)_: 

:clock10: **Incremental building**: average incremental building time is not available, there is no data for this repository.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[5913798](https://github.com/sdwheeler/PowerShell-Docs/pull/3598/commits/59137988d56cadbdfe81f55ff06950452fbc3d04)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/3.0/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-3.0&branch=pr-en-us-3598) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/4.0/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-4.0&branch=pr-en-us-3598) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/5.0/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-5.0&branch=pr-en-us-3598) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/5.1/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-5.1&branch=pr-en-us-3598) (powershell-5.1) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C24%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901241547528144-3598%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=NPyPrgdMjLtnH76j%2Bueb7SKlMk1oXWL1PYS3QzZd7po%3D&st=2019-01-24T15%3A56%3A38Z&se=2019-02-24T16%3A01%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/3.0/Microsoft.PowerShell.Management/Test-Path.md (hunk 1 above)
Comment (sdwheeler) [reply reference/3.0/Microsoft.PowerShell.Management/Test-Path.md]: fixed

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[c25e8fa](https://github.com/sdwheeler/PowerShell-Docs/pull/3598/commits/c25e8fa8d6505e1600414b2f655c701e04f7e7f3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/3.0/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/3.0/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-3.0&branch=pr-en-us-3598) (powershell-3.0) |
[reference/4.0/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/4.0/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-4.0&branch=pr-en-us-3598) (powershell-4.0) |
[reference/5.0/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/5.0/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-5.0&branch=pr-en-us-3598) (powershell-5.0) |
[reference/5.1/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/5.1/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-5.1&branch=pr-en-us-3598) (powershell-5.1) |
[reference/6/Microsoft.PowerShell.Management/Test-Path.md](https://github.com/sdwheeler/PowerShell-Docs/blob/sdw-pr3537-ext/reference/6/Microsoft.PowerShell.Management/Test-Path.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.management/test-path?view=powershell-6&branch=pr-en-us-3598) (powershell-6) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C24%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201901241627063755-3598%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Y9oYlQL5QgMSgDDCNDecWm8iIfF3jExgpyLB6ul7IQY%3D&st=2019-01-24T16%3A35%3A59Z&se=2019-02-24T16%3A40%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**